### DISCUSSION TITLE
styled components jsxStyleProps

### DISCUSSION THREAD

**Discussion by EpsilonEpsilon** [chakra-ui/panda#2668] [Q&A] [answered]
Hi, I'm new to Panda CSS, and while reading the docs, I found this example:
```
const Page = ()=>{
    return (
        <styled.button marginTop="40px">

        </styled.button>
    )
}
```

But it doesn't work for me. I'm getting the following error:
```
TS2786: 'styled.button' cannot be used as a JSX component.   Its type 'StyledComponent<"button">' is not a valid JSX element type.     Type 'StyledComponent<"button">' is not assignable to type '(props: any, deprecatedLegacyContext?: any) => ReactNode'.       Type '(props: ComponentProps<"button">) => Element' is not assignable to type 'ReactNode'.
```
This one also doesn't work:
```
const Container = styled("div", {
    base:{
        bg:"red",
    }
})
TS2554: Expected 1 arguments, but got 2.

```
<img width="804" alt="image" src="https://github.com/chakra-ui/panda/assets/70793170/b540199f-d78c-4c43-bc68-3567722124ca">

And something like this is working...
<img width="629" alt="image" src="https://github.com/chakra-ui/panda/assets/70793170/7633f742-6aef-43d3-b8d7-2dc1e3cc102c">

Here is my Panda CSS configuration:

```
import { defineConfig } from "@pandacss/dev";

export default defineConfig({
  // Whether to use css reset
  preflight: true,

  // Where to look for your css declarations
  include: ["./src/**/*.{js,jsx,ts,tsx}", "./pages/**/*.{js,jsx,ts,tsx}"],

  // Files to exclude
  exclude: [],

  // Useful for theme customization
  theme: {
    extend: {},
  },

  // The output directory for your css system
  outdir: "styled-system",

  // The JSX framework to use
  jsxFramework: "react",
  jsxStyleProps:"all",
  // The CSS Syntax to use to use
  syntax: "template-literal",
});
```

I would be glad to get any help with this.

**Comment by astahmer** [comment, marked as answer]
JSX style props are not available when using `syntax: "template-literal"
https://panda-css.com/docs/concepts/template-literals#getting-started

**Reply by EpsilonEpsilon** [reply]
but what about this https://panda-css.com/docs/concepts/style-props, 
<img width="802" alt="image" src="https://github.com/chakra-ui/panda/assets/70793170/7dab28bf-7cba-4d70-8bbd-18233608f121">

**Reply by astahmer** [reply]
this works with the default `syntax` config, e.g ` syntax: "object-literal",` https://panda-css.com/docs/references/config#syntax
